pyproject.toml

```diff
@@ -14,6 +14,7 @@ dependencies = ["lightgbm>=4.5.0, <5",
                 "cffi>=1.17.1, <2",
                 "scipy>=1.14.1, <2",
                 "matplotlib>=3.9.2, <4",
```

Comment:
change dependencies to have constant versions as in week 2 branch of the course
otherwise model deployment for model with feature lookup would fail